src/redisai.c

```diff
@@ -1036,62 +1036,66 @@ int RedisAI_ScriptSet_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv
 
 #define EXECUTION_PLAN_FREE_MSG 100
 
-#define REGISTER_API(name, registerApiCallback) \
-  if(registerApiCallback("RedisAI_" #name, RAI_ ## name)){\
+#define REGISTER_API(name, ctx) \
+  RedisAI_ ## name = RAI_ ## name;\
+  if (RedisModule_ExportSharedAPI) {\
+    if (RedisModule_ExportSharedAPI(ctx, "RedisAI_" #name, RAI_ ## name) != REDISMODULE_OK) {\
       printf("could not register RedisAI_" #name "\r\n");\
```

Comment:
We can now use RedisModule_Log cause we have the ctx.

Reply:
That's right!